amt/schema/instrument.py

```diff
@@ -11,6 +11,7 @@ class Owner(BaseModel):
 class InstrumentTask(BaseModel):
     question: str
     urn: str
+    type_: list[str] = Field(default=[], alias="type")
```

Comment:
why does the name have an underscore in it?

Reply:
Because my editor highlighted type differently than other variables, so I wrongly assumed it was a reserved keyword, which it is apparently not.